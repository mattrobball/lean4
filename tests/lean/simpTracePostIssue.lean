def f (x : Nat) := 0 + x

@[simp ↓]
theorem f_eq : f x = x := by simp [f]

set_option tactic.simp.trace true in
example : f (f x) = x := by
  simp

example : f (f x) = x := by
  simp only [↓f_eq]
