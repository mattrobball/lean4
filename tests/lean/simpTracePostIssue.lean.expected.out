Try this: simp only [â†“f_eq]
